### PR TITLE
MCFS-868 Price Request/Response change proposal to support multi data-source

### PR DESCRIPTION
**Pull request by niloufarmp**
Price Request/Response change proposal to support multi data-source

Please review the changes I made in pricing_and_booking_nilou.html and compare it to pricing_and_booking.html

### CODE REVIEW

**Comment by mcclymont-k** [comment]
I dont think I can look at this locally - can we just push it to master so we can look at it more easily there? 
@Iceman997  @niloufarmp

**Review by andrescollart — APPROVED** [review]
Approved

**Review by Iceman997 — APPROVED** [review]

File: tn_api_pricing_booking_spec_nilou.yml [hunk 1]
```diff
@@ -0,0 +1,874 @@
...
+                format: 'yyyy-MM-ddTHH:mm:ssZ'
+                description: Departure time of the flight
+                required: true
+                example: "2019-04-07T13:15:00.000+02:00"
+            arrival_time:
+                type: DateTime
+                description: Arrival time of the flight
+                format: 'yyyy-MM-ddTHH:mm:ssZ'
+                example: "2019-04-07T15:10:00.000-04:00"
+            branding_identifier:
```
Comment (Iceman997) [inline tn_api_pricing_booking_spec_nilou.yml]:
```suggestion
            brand_identifier:
```

Comment (mcclymont-k) [reply tn_api_pricing_booking_spec_nilou.yml:511]: Agreed - ill change it on MCFS to match this

**Review by mcclymont-k — APPROVED** [review]

**Pull request merged**